snippets/otel-collector-custom-resource.adoc

```diff
@@ -0,0 +1,46 @@
+// :_mod-docs-content-type: SNIPPET
+// Text snippet included in the following modules:
+//
+// * otel-install-cli.adoc
+// * otel-install-web-console.adoc
+.Example `OpenTelemetryCollector` CR
+[source,yaml]
+----
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: otel
+  namespace: <project_of_opentelemetry_collector_instance>
+spec:
+  mode: deployment
+  config: |
+    receivers: # <1>
+      otlp:
+        protocols:
+          grpc:
+          http:
+      jaeger:
+        protocols:
+          grpc: {}
+          thrift_binary: {}
+          thrift_compact: {}
+          thrift_http: {}
+      zipkin: {}
+    processors: # <2>
+      batch: {}
+      memory_limiter:
+        check_interval: 1s
+        limit_percentage: 50
+        spike_limit_percentage: 30
+    exporters: # <3>
+      debug: {}
+    service:
+      pipelines:
+        traces:
+          receivers: [otlp,jaeger,zipkin]
+          processors: [memory_limiter,batch]
+          exporters: [debug]
+----
+<1> For details, see the "Receivers" page.
+<2> For details, see the "Processors" page.
+<3> For details, see the "Exporters" page.
```

Comment:
Couple of quick questions. 
I see there is no callout in the earlier yaml. Since it's added as a snippet now with callouts to reference pages, are we looking at reusing it in more places?
I just wanted to know the intent of this change.

Reply:
@snarayan-redhat, yes, I am reusing this snippet in two procedures that are on the same page:
https://79138--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-installing.html#installing-otel-by-using-the-web-console_install-otel
https://79138--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-installing.html#installing-otel-by-using-the-cli_install-otel